guides/ReleaseNotes.md

```diff
@@ -1,6 +1,15 @@
 # Release Notes
 
 
+## New Endpoints: [Members API](members/members)
+The [Members API](https://dev.wix.com/api/rest/members/members) now includes three new endpoints:
++ [MemberCreated](https://dev.wix.com/api/rest/members/members/member-created-webhook) is triggered when a member is created.
++ [MemberDeleted](https://dev.wix.com/api/rest/members/members/member-deleted-webhook) is triggered when a member is deleted.
++ [MemberUpdated](https://dev.wix.com/api/rest/members/members/member-updated-webhook) is triggered when a member is updated.
```

Comment:
relative links - you can delete `https://dev.wix.com/api/rest/` from each of these